bgpd/bgp_main.c

```diff
@@ -463,7 +465,7 @@ int main(int argc, char **argv)
 			break;
 		}
 		case 'l':
-			bgp_address = optarg;
+			listnode_add(bgp_addresses, optarg);
```

Comment:
Shouldn't we check if we are not duplicating addresses here?

Reply:
You are right. It would be problematic if an address is specified more than once. I changed the code so duplicated addresses are ignored.